explainer.md

```diff
@@ -591,6 +591,90 @@ vrSession.addEventListener('resetpose', vrSessionEvent => {
+
+```js
+// Requests that a session be created if this pages was navigated to while the
+// previous page was viewing WebVR content.
+vrDevice.requestSession({ exclusive: true, waitForNavigate: true }).then(OnSessionStarted);
+```
+
+The UA should provide a visual transition between the two pages. At minimum the previous page should fade to black and the new one should fade in from black.
+
+For convenience both the `waitForNavigate` and `waitForActivate` options can be combined when calling `requestSession`. When both options are set the promise should only reject when both criteria are no longer valid. In other words, calling `requestSession` with the options `{ waitForNavigate: true, waitForActivate: true }` will not cause the promise to reject if the page is not VR navigable, but will instead continue to wait for an activation event to occur.
+
+```js
+// Requests that a session be created if this pages was navigated to while the
+// previous page was viewing WebVR content OR when the device is activated.
+vrDevice.requestSession({
+    exclusive: true,
+    waitForActivate: true,
+    waitForNavigate: true 
+  }).then(OnSessionStarted);
+```
+
+### Cancelling deferred session requests
+
+Because deferred session requests may be outstanding for a long time it's possible that a previously made request could become undesirable. And example would be if an associated piece of content has been scrolled off the page, which would make it confusing for the page to then began presenting that content in VR upon activation. In these cases the deferred request can be cancelled.
+
+To make a session request cancellable an [`AbortController`](https://dom.spec.whatwg.org/#interface-abortcontroller) is created. The controller's `signal` attribute is then passed into the `requestSession` call with the `signal` option to indicate that the request can be cancelled. Calling the `AbortController`'s `abort` method will cancel all outstanding promises that the signal was provided to.
+
+```js
+// Requests that a session be created when the device is activated.
+let abortController = new AbortController();
+vrDevice.requestSession({
+    exclusive: true,
+    waitForActivate: true,
+    signal: abortController.signal })
+  .then(OnSessionStarted)
+  .catch((reason) => {
+    if (abortController.signal.aborted) {
+      console.log("Session request was canceled by the page.");
+    } else {
```

Comment:
nit: this conditional logic doesn’t necessarily mean the “system” rejected it. there could’ve been an error thrown in `OnSessionStarted`.